u3-views/features/videoController.js

```diff
@@ -0,0 +1,20 @@
+const { getVideo,getSuggestions } = require('../_services/fakeapi')
+
+const videoDetails1 = (req, res) => {
+  const videoId = req.params.code
+  const video = getVideo(videoId)
+  const teases = getSuggestions(6)
+res.render('video-player', { video: video, teases: teases })
+}
+
+const videoDetails2 = (req, res) => {
+  const videoId = req.params.code
+  const video = getVideo(videoId)
+  const teases = getSuggestions(30)
```

Comment:
คุณสับสนตัวเองด้วยการตั้งชื่อ `videoDetails2` นี้ ควรเป็น `videoList` หรือ `videoHome` หน้านี้แสดงรายการวิดีโอจำนวนมาก ไม่มี "รหัสวิดีโอ" คุณควรโหลดข้อมูลสำหรับวิดีโอหลายรายการ:

You have confused yourself by naming this `videoDetails2`. It should be `videoList` or `videoHome`. This page shows a list of many videos. There is no `videoId`. You should load data for multiple videos:
```js
const videos = getSuggestions(30)
res.render('video-home', { videos: videos })
```